build/dotnet/dotnet-install.ps1

```diff
@@ -45,20 +45,24 @@
 .PARAMETER Verbose
```

Comment:
These changes reflect updates and bug fixes to the dotnet-install.ps1 script in https://github.com/dotnet/cli. The layout of assets on dotnetcli.azureedge.net is slightly different for the MSBuild SDK than it was for project.json SDK.